beard/utils/strings.py

```diff
@@ -13,10 +13,13 @@
 
 """
 
-import unicodedata
+import re
 import sys
+import unicodedata
 
 IS_PYTHON_3 = sys.version_info[0] == 3
```

Comment:
I must have missed it in the previous review, but why not simply use `six.u()` instead of this trick?